lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*' || node.args[1].op === '/') {
+    return false;
+  }
+  const args = node.args;
+  if (args.some(n=> Node.PolynomialTerm.isPolynomialTerm(n))) {
+    return false;
+  }
+  if (!args.every(n => Node.ConstantTerms.isConstantTerm(n))) {
```

Comment:
hmmmmmm I think we make this a lot simpler!

I see you're basing this around `PolynomialTerm` which is definitely an intuitive way to think about this

`PolynomialTerm` is used in a lot of places and has coefficients, whereas I'm not sure if `ConstantTerm` is as clear of a concept and I don't think it needs to have a coefficient (`2*5^2` is just a multiplication of two things, whereas `2x^2` is more of a single unit, if that makes sense)

my recommendation is to have a helper function that returns true if the node is a constant with the right value or an exponent node where the base is a constant with the right value - that's all we need right?

let me know what ya think - it'd be a bit of a redesign from your current solution